### DISCUSSION TITLE
Reference manager integration (e. g. Zotero)

### DISCUSSION THREAD

**Discussion by yann1cks** [athensresearch/athens#1198] [Feature Requests]
Athens is a tool for researchers and reference management is essential in many areas of research. Zotero is one of the most famous reference managers (also the one I am using) and open source. So its a perfect fit with Athens, but there are many other reference managers as well. 

I have no idea how interesting such a feature is, but many other "note-taking-apps" for academia offer some support for reference managers. There is long and ongoing discussion on the [Obsidian forum](https://forum.obsidian.md/t/zotero-best-practices/164/57), a [picker ](https://github.com/AlexanderSavenkov/Zotero-Obdisian-Picker-Windows)and a [plugin](https://github.com/hans/obsidian-citation-plugin). Zettlr also [has an integration](https://docs.zettlr.com/en/academic/citations/), which is one of their USPs. Furthermore there is a [Roam Export Plugin](https://github.com/melat0nin/zotero-roam-export) and a [Roam Plugin](https://alix-lahuec.gitbook.io/zotero-roam/getting-started/prereqs) that uses Zoteros API. There are integrations [for VS Code, Scrivener and other tools](https://retorque.re/zotero-better-bibtex/citing/cayw/). There are also many questions on Reddit. As far as I know Athens doesnt have any support for plugins and I believe native support would be great.

There are two different ways how Athens can access Zotero's references. The first is using the Better BibTex plugin and create a BibTex file that is automatically updated (used by Zettlr or the Obsidian Plugin). This can also be used to connect to other reference managers. The alternative is using the Zotero API as the Roam plugin does, which offers more features. 

I have been thinking about a minimal set of features for such a plugin:
- Create a page in Athens for a reference, that includes metadata and a URI to the file. Additionally importing notes from Zotero would be great (there a fundamental changes in the current Zotero Beta). 
- in-text reference which would automatically create a page in Athens based on the reference.

These two features would fulfil my needs but I am sure, that there are other important features. The two features I am describing are actually similar to the features of the Roam and Obsidian plugins.

**Comment by BradKML** [comment]
From https://github.com/athensresearch/athens/issues/1621

Requirements:
- Read Data from Zotero's CJSON file (with Better BibTex plugin) and add it to the knowledge base
- Able to click into the Book/Paper page and access the files from Zotero (Local Instance)
- View the papers DOI and metadata directly from the App
- Make notes with keywords/[[pages]]/alias for referencing issues

Current solutions partially provided by other applications:
- https://github.com/aljedaxi/logseq-zotero/
- https://discuss.logseq.com/t/using-zotero-with-logseq-via-obsidian/1296
- https://alix-lahuec.gitbook.io/zotero-roam/
- https://github.com/melat0nin/zotero-roam-export
- https://www.cortexfutura.com/zotero-and-roam-research/
(Issue: Foam requires VSCode, whilst LogSeq does not have Zotero Local Instance support)

**Comment by jellehuisman** [comment]
Zotero integration is mission critical for me, so I hope this will be implemented.
